src/Ursa.Themes.Semi/Controls/IconToggleSplitButton.axaml

```diff
@@ -0,0 +1,47 @@
+﻿<ResourceDictionary
+    xmlns="https://github.com/avaloniaui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:u="https://irihi.tech/ursa">
+
+    <ControlTheme x:Key="{x:Type u:IconToggleSplitButton}" TargetType="u:IconToggleSplitButton" BasedOn="{StaticResource BaseIconButton}">
+        <Setter Property="Template" Value="{StaticResource IconSplitButtonTemplate}" />
+        <Style Selector="^:not(:checked) /template/ u|IconButton">
+            <Setter Property="Theme" Value="{StaticResource {x:Type u:IconButton}}" />
+            <Style Selector="^#PART_SecondaryButton">
```

Comment:
Couldn't find a cleaner way to fix the secondary button's arrow's colors. My attempts to style the `ContentPresenter` within it led to weird results, binding to `$parent[u:IconToggleSplitButton]` didn't work either. Maybe there's a better way, but this should work.